Update upgrade-19.mdx with gas configuration check (#21632)
ethereum-optimism/optimism: docs/public-docs/notices/upgrade-19.mdx

  Make the following checks to verify that your node is properly configured:
    * The configurations will be logged at startup
    * Check that the Karst time is set to its correct activation timestamp

>     * Check that `keep_karst_upgrade_gas` is set to the expected value (see [this section](#gas-limit-action-required-for-affected-chains))

Author: ```suggestion
    * Check that `keep_karst_upgrade_gas` is set to true if appropriate (see [this section](#gas-limit-action-required-for-affected-chains))
```

Author: Since op-node will not log this at all if it is unset or set to `false`.

>     * Check that `keep_karst_upgrade_gas` is set to true if appropriate (see [this section](#gas-limit-action-required-for-affected-chains))

Author: ```suggestion
    * In the `op-node` startup logs, check that `keep_karst_upgrade_gas` is set to true if appropriate (see [this section](#gas-limit-action-required-for-affected-chains))
```
